cmd/listen-notification-handlers.go

```diff
@@ -139,6 +139,7 @@ func (api objectAPIHandlers) ListenNotificationHandler(w http.ResponseWriter, r
 		return rulesMap.MatchSimple(ev.EventName, ev.S3.Object.Key)
 	})
 
+	values.Set(peerRESTListenBucket, bucketName)
```

Comment:
set this value only if `bucketName != ""`

Reply:
I just pushed the proposed change. Let me know if there is anything else I can do 😄